### DISCUSSION TITLE
A320 FBW and autobrake

### DISCUSSION THREAD

**Discussion by cromsan** [flybywiresim/aircraft#2397] [Help]
Since last update MSFS2020 (10/12/20), before landing with A320 FBW development version, I can not put the autobrakes in any positión; so it is very difficult to stop the aircraft on the runway when you land.

**Comment by MapleNAV-TCASFAULT** [comment]
You're running the custom FBW version, or just the development version? We do not provide support for the custom FBW version as it is a WIP. But if you're running the actual dev version, how are you arming your autobrake, via switch or mouse click?

**Reply by cromsan** [reply]
I am running the development version. I am arming the development version with mouse click.

**Reply by MapleNAV-TCASFAULT** [reply]
Does this only happen with the mod? And is it frequently happening? Try reinstalling the mod and/or the default a320 in content manager in MSFS. If it continues to persist, place a GitHub issue if it has not been reported it. :)

**Reply by mga010** [reply]
The auto-brakes are armed LO or HI with two buttons. There is no "position". Are you talking about the ground spoilers?

**Comment by cromsan** [comment]
MapleNAV-TCASFAULT, I only has fly the A320 FBW, I do not know with the mod. And is the first time it has happen, after last update 10/12/20.

**Reply by MapleNAV-TCASFAULT** [reply]
The mod is FBW lol. Make sure you also downloaded the mod through the the installer.
Link: https://github.com/Externoak/A32NX-installer/releases press the A32NX_Downloader.zip
